models/src/main/java/com/vimeo/networking2/Privacy.kt

```diff
@@ -0,0 +1,39 @@
+package com.vimeo.networking2
+
+import com.vimeo.networking2.enums.CommentValue
+import com.vimeo.networking2.enums.EmbedValue
+import com.vimeo.networking2.enums.EmbedValue.UNKNOWN
+import com.vimeo.networking2.enums.ViewValue
+
+data class Privacy(
+
+    /**
+     * Whether other users can add the user's videos.
+     */
+    val add: Boolean,
```

Comment:
should have a default value and should be nullable according to the pattern used in other classes.